cluster-autoscaler/processors/processors.go

```diff
@@ -77,7 +77,6 @@ func DefaultProcessors() *AutoscalingProcessors {
 		NodeInfoProcessor:          nodeinfos.NewDefaultNodeInfoProcessor(),
 		NodeGroupConfigProcessor:   nodegroupconfig.NewDefaultNodeGroupConfigProcessor(),
 		CustomResourcesProcessor:   customresources.NewDefaultCustomResourcesProcessor(),
-		TemplateNodeInfoProvider:   nodeinfosprovider.NewDefaultTemplateNodeInfoProvider(),
```

Comment:
Why did you remove this line?

Reply:
I don't think it is default processors after introducing an ttl time parameter. In order to not pass a ttl time to DefaultProcessors() function I initialise TemplateNodeInfoProvider separately in main.go

Reply:
I understand, but now you've changed the default implementation so that it always has a nil `TemplateNodeInfoProvider`, so anyone who used to rely on this will suddenly get a nil pointer (and most likely `panic` down the road). I suggest either keeping it initialized with some default TTL value (like 10 years) or mention breaking change in a release note.

Reply:
Got it, thanks. I returned TemplateNodeInfoProvider.